cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/components/imageCarousel.isml

```diff
@@ -17,6 +17,7 @@
                 <isloop items="${cldImageURLsList}" var="image" status="loopStatus">
                     <div class='carousel-item <isif condition="${loopStatus.index == 0}">active</isif>'>
                         <img class="d-block img-fluid ${image.isResponsive ? 'cld-responsive' : ''}"
+                            src="${image.url}"
```

Comment:
Was this added intentionally?

Reply:
yes, that's correct

Reply:
the src attribute should only be defined when not using the Responsive JS, as done 2 lines below. We've aligned it for all images through the cartridge

Reply:
when we enable isResponsive true for PDP, it doesn't work in that case

Reply:
so we need to find out the root cause. Most likely the .responsive() is not called. Can you share a link demoing that this doesn't work?

Reply:
https://zzgt-004.dx.commercecloud.salesforce.com/s/RefArch/tweed-cowl-neck-sweater/25493689M.html?lang=en_US

Reply:
Per the documentation https://cloudinary.com/documentation/responsive_client_side_js#step_2_set_the_img_tag_parameters the transformation should have 'c_limit,w_auto'
In addition, I see an image

Reply:
cool, thanks a lot, I will make the requested changes

Reply:
I've made the changes but it seems not working, it breaks all the carousel images except the first, it loads the first image but when you change the image, it breaks

Reply:
I get 4 images correct and 2 images 404 (nothing related to responsiveness). Can you test another PDP page?

Reply:
I was trying to add a fix, that's why you see the images. When we change the image on PDP it never calls the responsive method, I have added a logic to call the responsive method on every image change.

Reply:
@asad-rafter I looked at the page layout and saw that the other items on the gallery have display:block, so the browser doesn't calc their layout dimensions, and this is why the responsive JS cannot apply a width on them.

Are we flexible with slightly changing the CSS of the widget?

Another request is to add the cloud name to the instantiation as follows: cloudinary.default.Cloudinary.new({cloud_name: <cloud name>})

Reply:
tried with CSS but there was no luck, we need to trigger the responsive method as I have already done in my latest commit

Reply:
I fully agree. let's wrap up here